src/app/guest-book/author-details/author-details.component.html

```diff
@@ -0,0 +1,14 @@
+<h2 mat-dialog-title>Author's Details</h2>
+<mat-dialog-content class="dialog-content">
+  <div class="profile">
+    <div class="avatar">
+      <img mat-card-image alt="Avatar" [src]="generateGravatarSource()" />
+    </div>
+    <div class="author-name">
+      <p>{{ authorEmail }}</p>
+    </div>
+  </div>
+  <div class="biography">
+    <p>{{ generateBio() }}</p>
```

Comment:
This is not a good practice to use getters / getter like functions - they are triggered too often (try to add console log to function)

Preferred way is to use pipes